c10/core/impl/LocalTensorTypeSet.h

```diff
@@ -0,0 +1,22 @@
+#include <c10/core/TensorTypeSet.h>
+
+// TLS management for TensorTypeSet
```

Comment:
any reason you didn't call this TLSTensorTypeSet?  I feel like if that's the first comment, it should probably be in the name.

Reply:
Because it is not "Thread local state tensor type set", it is "thread local tensor type set". But ThreadLocalTensorTypeSet is long so I dropped the Thread. I can rename it if you want.